throw new Error("任务1失败...")